src/api/app/views/layouts/webui/responsive_ux/_bottom_navigation.html.haml

```diff
@@ -11,15 +11,19 @@
           %i.fas.fa-tasks.fa-lg
           - if tasks.positive?
             .badge.badge-primary.align-text-top= tasks
-      %li.nav-item.border-left.border-gray-500
-        = link_to('javascript:void(0)', class: 'nav-link text-light', alt: 'Menu', data: { toggle: 'offcanvas' }) do
-          %i.fas.fa-bars.fa-lg
+      - if content_for?(:contextual_actions)
+        %li.nav-item.border-left.border-gray-500
+          = link_to('javascript:void(0)', class: 'nav-link text-light', alt: 'Contextual Actions', data: { toggle: 'contextual-actions' }) do
+            %i.fas.fa-th.fa-lg
```

Comment:
What about using `ellipsis-v` instead? I think nowadays most of the people is used to the three dots icon to find extra actions on the current element. It would look like:

![3dots](https://user-images.githubusercontent.com/2581944/76230944-c44dd000-6224-11ea-9e5d-2028e730fb99.png)

Reply:
Good idea, I've changed the icon for both navigation (small and large devices).